packages/core/saved-objects/core-saved-objects-migration-server-internal/src/next.ts

```diff
@@ -312,14 +315,16 @@ export const next = (
   transformRawDocs: TransformRawDocs,
   readyToReindex: WaitGroup<void>,
   doneReindexing: WaitGroup<void>,
-  updateRelocationAliases: WaitGroup<Actions.AliasAction[]>
+  updateRelocationAliases: WaitGroup<Actions.AliasAction[]>,
+  hashToVersionMap: Record<string, string>
```

Comment:
NIT: Why is this `hashToVersionMap` passed to `nextActionMap` instead of being added to the state/baseState?

Reply:
It is not used by the model, and it does not change throughout the flow.
I thought having to clone it every time we create a new state object was costly and unnecessary.
This probably falls under the _state vs context_ discussion.